augur/export.py

```diff
@@ -27,6 +29,10 @@ def tree_to_json(node, fields_to_export = [], top_level = [], div=0):
             if hasattr(node, fname):
                 val = node.__getattribute__(fname)
 
+        # shadow clade by strain. clade is deprecated and will be removed.
+        if field == "clade"
```

Comment:
Missing a colon here. The "clade" attribute is only deprecated for auspice, but not for augur, right?